### ISSUE TITLE
Since you spend so much time teaching, do you still feel like you're learning?

### ISSUE THREAD

**Issue by jLouzado** [kentcdodds/ama#441]
The question is in the title, really but let me explain.

I'm a senior-ish dev where I work and I find myself spending a lot of time teaching. That's not really a bad thing except I feel like I'm repeating the same lessons over and over again and off late I've begun to feel like I'm investing a lot in other people and I'm not sure if my learning is suffering of not.

Thanks for your time and for all you do for the community. :heart:

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=4Yq6PO0r-fU&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
